tests/integration/tier1/destructive/proxy-conversion/test_proxy_conversion.py

```diff
@@ -91,36 +92,29 @@ def setup_rhsm(shell):
         ).returncode
         == 0
     )
-    if SYSTEM_RELEASE_ENV in ("centos-7", "oracle-7"):
-        assert (
-            shell(
-                f"curl -o /etc/yum.repos.d/client-tools.repo https://cdn-public.redhat.com/content/public/repofiles/client-tools-for-rhel-7-server.repo \
-                    --proxy http://{TEST_VARS['PROXY_SERVER']}:{TEST_VARS['PROXY_PORT']}",
-                silent=True,
-            ).returncode
-            == 0
-        )
-    else:
-        assert (
-            shell(
-                f"curl -o /etc/yum.repos.d/client-tools.repo https://cdn-public.redhat.com/content/public/repofiles/client-tools-for-rhel-8.repo \
-                    --proxy http://{TEST_VARS['PROXY_SERVER']}:{TEST_VARS['PROXY_PORT']}",
-                silent=True,
-            ).returncode
-            == 0
-        )
-        # On CentOS 8.5 we need to replace the $releasever in the url to 8.5,
-        # otherwise the dnf will complain with dependency issues.
-        if "centos-8" in SYSTEM_RELEASE_ENV:
-            shell(r"sed -i 's#\$releasever#8.5#' /etc/yum.repos.d/client-tools.repo")
+
+    client_tools_repo = ""
+    if re.match(r"^(centos|oracle)-7", SYSTEM_RELEASE_ENV):
```

Comment:
maybe we could use the `SystemInformationRelease.version.major == X` here instead of the `SYSTEM_RELEASE_ENV`. But that is just a thought, no need to change it

Reply:
That's valid, I'll do that.